src/main/java/frc/robot/Constants.java

```diff
@@ -123,6 +123,42 @@ public static class Auto {
 
     /** The amount of time that we want to run the fire note command in auto. */
     public static final double FIRE_NOTE_FOR_TIME = 4.0;
+
+    /** Time in seconds that we delay our taxi in the delayed taxi auto. */
+    public static final double TAXI_DELAY_TIME = 10.0;
+
+    /** Time in seconds that we wait for FireNoteRoutine to end. */
+    public static final double DUMP_NOTE_WAIT_TIME = 4.0;
+
+    /** Time in seconds that we wait for PickupFromCenter to end. */
+    public static final double PICKUP_CENTER_WAIT_TIME = 5.0;
+
+    /** Time in seconds that we wait for TaxiLongAuto to end. */
+    public static final double TAXI_LONG_WAIT_TIME = 2.0;
+
+    /** Constants for the DriveToAmpBlue auto */
+    public static class DriveToAmpBlue {
+      /** The time for the robot to drive left towards the amp. */
+      public static final double LEFT_WAIT_TIME = 1;
+
+      /** The time for the robot to drive forwards towards the amp. */
+      public static final double FORWARD_WAIT_TIME = 0.5;
+    }
+
+    /** Constants for the DriveToAmpRed auto */
```

Comment:
Javadoc still says DriveToAmpRed

Reply:
resolved by me growing brain cells and pushing before requesting your review 👍